cmd/acraserver/acraserver.go

```diff
@@ -25,10 +25,15 @@ import (
 	"github.com/cossacklabs/acra/network"
 	"github.com/cossacklabs/acra/utils"
 	log "github.com/sirupsen/logrus"
+	"os/signal"
+	"time"
+	"errors"
 )
 
 // DEFAULT_CONFIG_PATH relative path to config which will be parsed as default
 var DEFAULT_CONFIG_PATH = utils.GetConfigPathByName("acraserver")
+var SignalsChannel = make(chan os.Signal)
```

Comment:
https://github.com/cossacklabs/acra/blob/master/cmd/utils.go#L37
We has signal handler that used in acraproxy/acraserver (added before your PR:). Better to use one implementation instead many in different places. Better to extend existing in this case imho.

Reply:
Refactored